packages/javascript/bh-shared-ui/src/providers/NotificationProvider/hooks.ts

```diff
@@ -23,6 +24,11 @@ import {
     type NotificationAction,
 } from './actions';
 
+export const PERSIST_NOTIFICATION: OptionsObject = {
```

Comment:
Moved this from jobs util module to notification provider to be with the notification hook that uses it.